aip/client-libraries/4223.md

```diff
@@ -0,0 +1,44 @@
+---
+id: 4223
+state: draft # TODO(noahdietz): Update pre-merge
+created: 2022-05-11 # TODO(noahdietz): Update pre-merge
+---
+
+# Client-side payload validation
+
+Client-side payload validation is the practice of inspecting the request payload
+for any misconfigurations (e.g. missing required fields, attempting to update an
+immutable field) prior to executing a network call with the payload. Typically,
+documentation communciates the expectations of the service with regards to each
+request field, including any requirements on presence or format. The
+`google.api.field_behavior` annotation defined in [AIP-203][] is the
+machine-readable form of this documentation that services use to communicate
```

Comment:
How about 
"machine-readable format that services use to document"

"Communicate" suggests the questions "communicate to whom? Why shouldn't the libraries use this communication?". "Document" arguably suggests more strongly it is intended for human consumption, even if via machine processing.

(not a huge issue, so feel free to ignore)

Reply:
love it, done!